version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "4.1.1"
+version in ThisBuild := "4.1.1-spark-3.0.1"
```

Comment:
@lucagiovagnoli what are thoughts w.r.t. just releasing this as a minor/patch upgrade rather than adding a suffix?

Reply:
the version may be related to the comment above